testplan/testing/multitest/base.py

```diff
@@ -857,6 +884,9 @@ def aborting(self):
 
     def _log_status(self, report, indent):
         """Log the test status for a report at the given indent level."""
+        passed = report.passed
+        if report.xfail:
+            passed = report.xfail['result'].capitalize()
```

Comment:
this is wrong, a `boolean` variable shouldn't be reassigned to a `str`. As above, just  pass the `report.status.tag` to the log function and let that deal with the different possible statuses, instead off overloading the `passed` variable.